pkg/runtime/runtime.go

```diff
@@ -371,10 +373,15 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	a.httpMiddlewareRegistry.Register(opts.httpMiddleware...)
 
 	go a.processComponents()
-	err = a.beginComponentsUpdates()
-	if err != nil {
-		log.Warnf("failed to watch component updates: %s", err)
+
+	if a.featureHotReloadingEnabled {
```

Comment:
Can we have an environment variable instead of an official feature flag? We know it does not work at all today, so why offer as a preview feature?

Reply:
Actually it's not even a preview feature (just a bool), but env var makes sense. Let me add it, thanks!

Reply:
@artursouza since it's unofficial, we don't need any documentation, right?

Reply:
I would not document this environment variable. We are just being extra cautious and not removing the code to being with.